### ISSUE TITLE
Dockerfile update 

### ISSUE THREAD

**Issue by adc0032** [lasseignelab/capture#7]
in the Dockerfile, `bin/docker/Dockerfile`, we would probably want to pull the `3_20` bioconductor container

**Comment by tsoelter** [comment]
I noticed the same thing yesterday when I built a new docker using that Dockerfile. Maybe we could use the `latest` tag, so we don't have to update continously?

**Comment by toniecrumley** [comment]
> I noticed the same thing yesterday when I built a new docker using that Dockerfile. Maybe we could use the `latest` tag, so we don't have to update continously? @toniecrumley

Would installing the latest version cause problems with reproducibility as the version changes rather than specifying a specific version?

I could change `cap new` to get the latest version at the time the project is created and put that into the Dockerfile.  ChatGPT recommended using this command to get the latest version.
```
curl -s https://hub.docker.com/v2/repositories/bioconductor/bioconductor_docker/tags | jq -r '.results[].name' | grep -E '^RELEASE_' | sort | tail -n 1

``` 
I get this result when I ran it `RELEASE_3_20-R-4.4.2`.

cc @adc0032 @tsoelter

**Comment by tsoelter** [comment]
I agree, I did not consider that causing reproducibility issues, but you are 100% correct. I think your proposed solution would be a great fix for this issue, since your result reflects the current/latest version. @toniecrumley
